Backend/Models/Timer.model.js

```diff
@@ -1,16 +0,0 @@
-const mongoose = require('mongoose');
-
-const TimerSchema = mongoose.Schema({
-   
-    taskId : String ,
```

Comment:
timer schema should have only task id and time info
it should not have, `taskName, userId `
those data you can be retreived from `TaskModel `

**no data duplication**